File: C/ascii.c
#include <stdio.h>

void main()
{
    char a, d;
    int b, c;
    printf("Enter a character: ");
    scanf("%c", &a);
    b = a;
    printf("ASCII value of %c is %i\n", a, b);
    printf("Enter an ASCII value: ");
    scanf("%i", &c);
    d = c;
    printf("Character with the ASCII value of %i is %c\n", c, d);
}

File: java Programming/1st Week/ascii.java
// 3. Write a program to find the ASCII code of a character

import java.util.Scanner;

public class ascii {
   public static void main(String[] args) {
      Scanner sc = new Scanner(System.in);
      System.out.print("Enter a character: ");
      char c = sc.next().charAt(0);
      sc.close();
      System.out.println("ASCII value of " + c + " is " + (int) c);
   }
}
